yarn.lock

```diff
@@ -12958,7 +12958,7 @@ immutability-helper@^3.1.1:
   resolved "https://registry.yarnpkg.com/immutability-helper/-/immutability-helper-3.1.1.tgz#2b86b2286ed3b1241c9e23b7b21e0444f52f77b7"
   integrity sha512-Q0QaXjPjwIju/28TsugCHNEASwoCcJSyJV3uO1sOIQGI0jKgm9f41Lvz0DZj3n46cNCyAZTsEYoY4C2bVRUzyQ==
 
-immutable@^3.8.1, immutable@^3.8.2:
+immutable@^3.8.1:
```

Comment:
immutable is still here because `@data-ui/event-flow` has it as a dependency. However, it's not actually used in that package at all, so treeshaking should remove it from the bundle